Sources/Apodini/Application/Application+FCM.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Application+FCM.swift
+//  
+//
+//  Created by Tim Gymnich on 30.12.20.
+//
+
+import FCM
+
+extension Application {
```

Comment:
I am not sure if this is already addressed by an other PR (might very well be out of scope for this PR) but we should think about moving this file and the FCM dependencies to the Notifications target.

Reply:
Yes, this also addressed by another branch. However, it requires to have a separate `VaporTarget` first before moving it to the `Notifications` target

Reply:
Sounds good 👍